### ISSUE TITLE
scaladoc3:  1-st sentence of class description is duplicated in generated scaladoc.

### ISSUE THREAD

**Issue by rssh** [lampepfl/dotty#12791]
## Compiler version

3.0.0

## Minimized code

source:  https://github.com/rssh/scala-gopher/blob/master/shared/src/main/scala/gopher/Select.scala

## Output

scaladoc:  http://rssh.github.io/scala-gopher/api/jvm/api/gopher/Select.html

## Expectation

The sentence "Organize waiting for read/write from multiple async channels" should be listed once.
Now it's duplicated in `documatableBried`  div and in `cover` div

(steps for reproducing 
```
sbt clone https://github.com/rssh/scala-gopher
cd scala-gopher
sbt previewSite
// navigate to js or jvm API and find select in browser.
```

**Comment by pikinier20** [comment]
Hey @rssh!
What version of Scaladoc did you use to generate documentation?
I've tried to reproduce this bug with code:
```scala
/** Organize waiting for read/write from multiple async channels
 *
 *  Gopher[F] provide a function `select` of this type.
 */
class Select[T](api: T)
```
and it looks good:
![image](https://user-images.githubusercontent.com/48246851/123598656-08d3f500-d7f5-11eb-90d0-8305898a67cf.png)

I remember that we had this bug in one of previous versions but it had been fixed.

**Comment by rssh** [comment]
What is bungled in 3.0.0
with 3.0.1-RC2 .... one moment --  looks fine.

So, looks like it was fixed between 3.0.0 and 3.0.1-RC2

**Comment by pikinier20** [comment]
Nice to hear that!
Looks like we can close this issue.
